CRM/Admin/Form/RelationshipType.php

```diff
@@ -157,20 +156,22 @@ public function postProcess() {
       $params['contact_type_a'] = $cTypeA[0];
       $params['contact_type_b'] = $cTypeB[0];
 
-      $params['contact_sub_type_a'] = $cTypeA[1] ? $cTypeA[1] : 'NULL';
-      $params['contact_sub_type_b'] = $cTypeB[1] ? $cTypeB[1] : 'NULL';
+      $params['contact_sub_type_a'] = $cTypeA[1] ? $cTypeA[1] : 'null';
```

Comment:
why are we changing to lower case here?

Reply:
because it wouldn't get through the api checks with caps - api permits 'null' not 'NULL' - I haven't see 'NULL' elsewhere